tests/pytests/test_info_modules.py

```diff
@@ -187,13 +188,82 @@ def test_redis_info():
     conn.execute_command('DEL', f'h{i}')
 
   # Force-invoke the GC
-  forceInvokeGC(env, 'idx')
-
-  # Wait for GC to finish
-  time.sleep(2)
+  forceInvokeGC(env)
 
   # Call `INFO` and check that the data is updated accordingly
   res = env.cmd('INFO', 'MODULES')
   env.assertGreater(res['search_bytes_collected'], 0)
   env.assertGreater(res['search_total_cycles'], 0)
   env.assertGreater(res['search_total_ms_run'], 0)
+
+
+def test_counting_queries(env: Env):
+  # Create an index
+  env.expect('FT.CREATE', 'idx', 'SCHEMA', 'n', 'NUMERIC').ok()
+  # Add some data
+  with env.getClusterConnectionIfNeeded() as con:
+    for i in range(10):
+      con.execute_command('HSET', i, 'n', i)
+
+  # Initiate counters
+  unique_queries_counter = 0
+  query_commands_counter = 0
+  def check_counters():
+    line_number = currentframe().f_back.f_lineno
```

Comment:
is this the line number in the test?

Reply:
Yes, the line number of the current call to this function. RLTest only shows the line number of the assertion, so we won't be able to know which call has failed without this count